frontend/src/app/components/wp-table/embedded/wp-embedded-table.component.ts

```diff
@@ -12,6 +12,7 @@ import {QueryFormResource} from "core-app/modules/hal/resources/query-form-resou
 import {QueryFormDmService} from "core-app/modules/hal/dm-services/query-form-dm.service";
 import {distinctUntilChanged, map, take, withLatestFrom} from "rxjs/operators";
 import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
+import {KeepTabService} from "core-components/wp-single-view-tabs/keep-tab/keep-tab.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/2a62a865520f2ce8e35cc0f9f8ab392f3dbd95dd/frontend/src/app/components/wp-table/embedded/wp-embedded-table.component.ts#L16"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>